examples/hello_world/shared/src/counter.rs

```diff
@@ -65,8 +65,10 @@ mod test {
 
         // Check update asked us to `Render`
         let actual_effect = &update.effects[0];
-        let expected_effect = &Effect::Render(RenderOperation);
-        assert_eq!(actual_effect, expected_effect);
+        match actual_effect {
+            Effect::Render(_) => assert!(true),
+            _ => assert!(false, "expected Render, got {:?}", actual_effect),
+        }
```

Comment:
This pattern feels a bit wordier then the original test, but not too bad?

Reply:
I agree, it's a little annyoing. I was looking at using something like https://github.com/poignardazur/assert_let to make that a bit more succinct.